vertical-pod-autoscaler/common/version.go

```diff
@@ -16,5 +16,26 @@ limitations under the License.
 
 package common
 
+// gitCommit is the commit used to build the VPA binaries, if available.
+// It is injected at build time.
+var gitCommit = ""
+
 // VerticalPodAutoscalerVersion is the version of VPA.
-const VerticalPodAutoscalerVersion = "1.2.0"
+const versionCore = "1.2.0"
+
+func VerticalPodAutoscalerVersion() string {
+	v := versionCore
+	if gitCommit != "" {
+		// NOTE: use 14 character short hash, like Kubernetes
+		v += "+" + truncate(gitCommit, 14)
+	}
+
+	return v
+}
+
+func truncate(s string, maxLen int) string {
+	if len(s) < maxLen {
+		return s
+	}
+	return s[:maxLen]
+}
```

Comment:
Most of this is copied from kind: https://github.com/kubernetes-sigs/kind/blob/1c5a56b30145d98a61c0538975ca0894ad65e0d0/pkg/cmd/kind/version/version.go#L42

Ideally I'd like to copy what Kubernetes does, but it's a larger project with a more complicated build process that I'm unfamiliar with